benchmarks/query_middleware.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+var http = require('http')
+var Benchmark = require('benchmark')
+var suite = new Benchmark.Suite()
+
+var sr = {}
+
+var middleware = require('../lib/middleware/query')()
```

Comment:
Gotcha. I guess I got confused because the benchmark is about different URLs, rather than the different options of the `query` middleware itself. For example, I think it would be more useful to test the 4 different query options we provide out of the box to each other, rather than testing urls to feed to just a single option (qs).